### PR TITLE
Updates to reflect v0.11 code

### PR DESCRIPTION
**Pull request by calmh**

### CODE REVIEW

File: BEPv1.md [hunk 1]
```diff
@@ -238,17 +252,19 @@ ClientVersion is "v0.7.2". The ClientVersion field SHOULD follow the
...
 standard.
 
-The Folders field lists all folders that will be synchronized
-over the current connection. Each folder has a list of participating
-Devices. 
+The Folders field lists all folders that will be synchronized over the
+current connection. Each folder has a list of participating Devices,
+Flags and Options. Currently no flags are defined so the field MUST be
+set to all zeroes. The Options field is implementation specific and
+described below.
```
Comment (AudriusButkevicius) [inline BEPv1.md]: This is not enforced though.

**Comment by AudriusButkevicius** [comment]
Looks good.

File: BEPv1.md (hunk 1 above)
Comment (calmh) [inline BEPv1.md]: Indeed. That's why it `MUST` be zero. :)

**Comment by calmh** [comment]
Cool. I'll take another look when I have more sense for detail and have had less wine, to make sure nothing important was forgotten. This time there are other people to give heads up! (Which is cool.)

File: BEPv1.md (hunk 1 above)
Comment (AudriusButkevicius) [inline BEPv1.md]: Yeah MUST which is not enforced is FINGERS CROSSED

**Pull request merged**